### DISCUSSION TITLE
Mirror the UR bar by default for osu!taiko gameplay.

### DISCUSSION THREAD

**Discussion by ianpep** [ppy/osu#23410] [Ideas] [answered]
Pretty self explanatory. For osu!taiko gameplay in stable, the UR bar is somewhat counter intuitive, with hits on the left being early, and late for the right. This does not match the gameplay however, since early hits would be right of the hit marker, and vice versa for late hits. Mirroring the UR bar by default would make adjusting offset much more understandable and easy for newer players to understand.

**Comment by ItsShamed** [comment]
Just horizontally flip the UR bar in the skin editor.

**Comment by peppy** [comment, marked as answer]
All depends on your perspective.
